### PR TITLE
feat(IT Wallet): [SIW-467] Add presentation data screen

### PR DESCRIPTION
**Pull request by LazyAfternoons**
## Short description
Depends on #5126
This PR adds the presentation data screen with the required mocked claims.

 

## List of changes proposed in this pull request
- Adds locales;
- Adds a divider in `ItwBulletList.tsx` for each section and refactors its padding to better match what's shown in the FIGMA;
- Adds `ItwPresentationDataScreen.tsx` as shown in the FIGMA. There are currently two issues, which will be addressed in different PRs. The first one is the TOS component, and the second one is the bottom sheet, which doesn't show the whole content but uses a ScrollView instead.

## How to test
Enable the experimental design system. Activate the wallet then add a credential from the wallet home screen. Long press the credential and test the flow.

### CODE REVIEW

**Review by hevelius — CHANGES_REQUESTED** [review]

File: ts/features/it-wallet/screens/presentation/crossdevice/new/ItwPresentationDataScreen.tsx [hunk 1]
```diff
@@ -0,0 +1,216 @@
...
+    navigation.navigate(ROUTES.MAIN, {
+      screen: ROUTES.MESSAGES_HOME
+    });
+  };
+
+  const ContentView = ({ decodedPid }: ContentViewParams) => (
+    <SafeAreaView style={IOStyles.flex}>
+      <ScrollView style={IOStyles.horizontalContentPadding}>
+        <VSpacer size={32} />
+        {/* SECOND HEADER */}
```
[the remark this answers is not in the document]
Comment (LazyAfternoons) [reply ts/features/it-wallet/screens/presentation/crossdevice/new/ItwPresentationDataScreen.tsx:83]: Definitely a good idea. That's a component which can be isolated and customized in the future based on the issuer.

File: ts/features/it-wallet/screens/presentation/crossdevice/new/ItwPresentationDataScreen.tsx [hunk 2]
```diff
@@ -0,0 +1,216 @@
...
+        <View
+          style={{
```
[the remark this answers is not in the document]
Comment (LazyAfternoons) [reply ts/features/it-wallet/screens/presentation/crossdevice/new/ItwPresentationDataScreen.tsx]: Addressed in [#3079949](https://github.com/pagopa/io-app/pull/5131/commits/3079949a1f246123a9f7a146ba9793c4a66cbea2).

File: ts/features/it-wallet/utils/mocks.ts [hunk 3]
```diff
@@ -155,10 +155,70 @@ const getRequestedClaims = (
...
 };
 
 export const rpMock: RpMock = {
-  organizationName: "eFarma"
+  organizationName: "eFarma",
+  requestedClaims: (decodedPid: PidWithToken) =>
+    getMultipleRequestedClaims(decodedPid),
+  optionalClaims: [
+    {
+      title: "Fornito da Credenziale 1",
```
[the remark this answers is not in the document]
Comment (LazyAfternoons) [reply ts/features/it-wallet/utils/mocks.ts]: I added the translation for the mock too in [#c3fae923](https://github.com/pagopa/io-app/pull/5131/commits/c3fae92359de3ece76dc6d5964ffe5b9b6353aea), if that makes sense. However we should add the `Provided by` part directly in the component which renders the claims list as I expect the credential not to carry the whole `Provided by issuer` string bur rather only the issuer. 
We can refactor this when we have a better idea of what's inside a credential.

**Review by hevelius — APPROVED** [review]
LGTM. Static code is ok. Not tested on real devices or simulators.

**Pull request merged**